Sources/GATTBootMouseInputReport.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  GATTBootMouseInputReport.swift
+//  Bluetooth
+//
+//  Created by Carlos Duclos on 6/19/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+
+/**
+ Boot Mouse Input Report
+ 
+ [Boot Mouse Input Report](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.boot_mouse_input_report.xml)
+ 
+ The Boot Mouse Input Report characteristic is used to transfer fixed format and length Input Report data between a HID Host operating in Boot Protocol Mode and a HID Service corresponding to a boot mouse.
+ */
+public struct GATTBootMouseInputReport: GATTCharacteristic {
+    
+    internal static let length = MemoryLayout<UInt8>.size
+    
+    public static var uuid: BluetoothUUID { return .bootMouseInputReport }
+    
+    public var value: UInt8
```

Comment:
Make it `var rawValue: UInt8`, make the type conform to `RawRepresentable` and `ExpressibleByIntegerLiteral`